src/components/status-image/status-image.css

```
@@ -1,4 +1,21 @@
 .StatusImage {
+    
     border: 1px solid white;
-    background-color: var(--color-busy);
+    height: 423px;
+    width: 414px;
+}
+
+img {
+    position: absolute;
+    max-height: 423px;
+    max-width: 414px;
+    left: 0;
+}
+
+#person {
```

Do NOT use ids if the styles are added in the global score. Use classes instead. Also add the class in the selector to avoid overriding other css rules.
```suggestion
.StatusImage img.person {
```